content/_partners/determining-your-assurance-level._en.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: partners/determining-your-assurance-level
+permalink: /partners/determining-your-assurance-level/
+title: >-
+    Determining your assurance level
+---
+
+## About assurance levels
+
+Assurance levels (also called “Service levels,” or “<a href="https://csrc.nist.gov/glossary/term/identity_assurance_level" class="external-link">Levels of Assurance</a>”) is a general term referring to the trustworthiness of a given transaction. Assurance levels are considered essential components of identity systems, due to the underlying complexity of identity verification processes. Generally, the higher the Service Levels, the greater the trustworthiness of the authentication and verification processes that occurred for a specific transaction and identity. 
+
+Assurance levels can be general or specific. <a href="https://pages.nist.gov/800-63-FAQ/#q-1" class="external-link">NIST’s 800-63 publication</a> previously was a monolithic Level of Assurance (LOA) in revisions 1 and 2. Revision 3, the current version, distinguishes between the level of confidence in an identity proofing process (IAL), authentication assurance level (AAL), and federation assurance level (FAL).
+
+**Examples:**
+
+<table class="usa-table margin-bottom-4">
+  <thead>
+    <tr>
+      <th scope="col"><b>Assurance Level Type</b></th>
+      <th scope="col"><b>Authentication Assurance Level (AAL)</b></th>
+      <th scope="col"><b>Identity Assurance Level (IAL)</b></th>
```

Comment:
It doesn't look like the `<b>` makes any difference here (i.e. the table's default styles already bold the heading text)? It also spares us a debate about [`<strong>` vs. `<b>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#b_vs._strong).

```suggestion
      <th scope="col">Assurance Level Type</th>
      <th scope="col">Authentication Assurance Level (AAL)</th>
      <th scope="col">Identity Assurance Level (IAL)</th>
```